### PR TITLE
Fixing issues in bookingSection

### PR DESCRIPTION
**Pull request by tramsi**

### CODE REVIEW

**Comment by tramsi** [comment]
https://tramsi.fitness.m3ntorship.com/
https://trello.com/c/S2FHXl3J/44-html-css-fix-booking-form-icons-name-input-type

File: dist/assests/css/booking.css [hunk 1]
```diff
@@ -66,19 +66,55 @@
...
+  background-color: white;
+}
+
+.booking-section__booking-name::placeholder {
+  font-weight: 700;
+  color: rgba(22, 20, 24, 0.2);
+}
+
+.booking-section__inputs__wrapper
+  input[type='date']::-webkit-calendar-picker-indicator {
```
Comment (AbudZohair) [inline dist/assests/css/booking.css:116]: Fix inner-spin-button 

tramsi date indicator

**Review by AbudZohair — CHANGES_REQUESTED** [review]

File: dist/assests/css/booking.css (hunk 1 above)
Comment (tramsi) [reply dist/assests/css/booking.css:116]: Done!

**Comment by AbudZohair** [comment]
Good job

**Review by AbudZohair — APPROVED** [review]

**Pull request merged**